test/lambdaconnect_model/utils_test.clj

```diff
@@ -0,0 +1,11 @@
+(ns lambdaconnect-model.utils-test
+  (:require [clojure.test :refer [deftest is testing]] 
+            [lambdaconnect-model.core :as mp] 
+            [clojure.pprint :as pprint :refer [pprint]]
+            [lambdaconnect-model.utils :refer [relevant-tags]]))
+
+(deftest relevant-tags-test
+  (let [contraint-to-be-tested '(and [= :tmp-field :tag0] (or [= :tmp-field :tag1] (and (or-join [= :tmp-field  :tag4] [= :tmp-field :tag2]) (not [= :tmp-field :tag3])))) 
```

Comment:
mógłbyś to jakoś sformatować żeby było bardziej czytelne? Chodzi mi o coś w stylu:
```
'(and [= :tmp-field :tag0]
      (or [= :tmp-field :tag1]
          (and (or-join [= :tmp-field :tag4] 
                        [= :tmp-field :tag2]) 
               (not [= :tmp-field :tag3]))))
```